docs/content/1.documentation/5.advanced/2.faq.md

```diff
@@ -376,3 +376,26 @@ Alternatively, you can use the `external` attribute on `NuxtLink` to set the nav
 ::alert{type="info"}
 ℹ Read more about it [here](https://github.com/Baroshem/nuxt-security/issues/228).
 ::
+
+### Running app with `--host` flag
+
+If you want to expose your app in local network to test it by using other devices you can do so by adding the following configuration:
+
+```ts
+security: {
+    headers: {
+      crossOriginEmbedderPolicy: process.env.NODE_ENV === 'development' ? 'unsafe-none' : 'require-corp', //https://github.com/Baroshem/nuxt-security/issues/101
+      strictTransportSecurity: false,
```

Comment:
strictTransportSecurity: true is better and will work

Reply:
Added